lib/Scan.js

```diff
@@ -46,68 +51,111 @@ Scan.prototype.exec = function (next) {
   var schema = Model$.schema;
   var options = this.options;
 
-  var scanReq = {
-    TableName: Model.$__.name
-  };
-
-  if(Object.keys(this.filters).length > 0) {
-    scanReq.ScanFilter = {};
-    for(var name in this.filters) {
-      var filter = this.filters[name];
-      var filterAttr = schema.attributes[name];
-      scanReq.ScanFilter[name] = {
-        AttributeValueList: [],
-        ComparisonOperator: filter.comparison
-      };
-
-      if(filter.values) {
-        for (var i = 0; i < filter.values.length; i++) {
-          var val = filter.values[i];
-          scanReq.ScanFilter[name].AttributeValueList.push(
-            filterAttr.toDynamo(val, true)
-          );
+  var scanReq = { };
+  var dbClient = null;
+
+/**/
+    function scanByRawFilter() {
+      var deferred = Q.defer();
+      var AWS = Model.$__.base.AWS;
+      AWS.config.update({ endpoint: 'http://localhost:8000' });
```

Comment:
Thanks. Let me check this to see what is going on at this point. I remembered I've already moved the AWS things into the model to keep consistency...